Assets/PatchKit Patcher/Scripts/Debug/UnityMessageWriter.cs

```diff
@@ -18,7 +18,9 @@ public UnityMessageWriter([NotNull] IMessageFormatter formatter)
 
         public void Write(Message message, MessageContext messageContext)
         {
-            var text = _formatter.Format(message, messageContext);
+            var patcher = Patcher.Instance;
+            var text = _formatter.Format(message, messageContext)
+                .Replace(patcher.AppSecret, patcher.TraceableAppSecret);
```

Comment:
`patcher.AppSecret` can be `null` at this point, and can cause some unwanted exception here.

Reply:
Done